BuildNotifications.Tests/ViewModels/Settings/Setup/ConnectionsSectionViewModelTests.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+using System.Linq;
+using System.Windows;
+using BuildNotifications.Core.Config;
+using BuildNotifications.Core.Plugin;
+using BuildNotifications.Services;
+using BuildNotifications.ViewModel.Settings.Setup;
+using NSubstitute;
+using Xunit;
+
+namespace BuildNotifications.Tests.ViewModels.Settings.Setup
+{
+    public class ConnectionsSectionViewModelTests
```

Comment:
Since there has been some problems in the connection settings in the past, I decided to test as much code as possible.
In general only testing the logic should suffice for us but testing ViewModels not wrong. That's one of the reasons to use ViewModels in the first place.

Reply:
Aight. Makes sense.